dashboard/app/views/layouts/_share_footer.html.haml

```diff
@@ -8,7 +8,7 @@
     className: 'dark',
     menuItems: [
       {
-        text: I18n.t('footer.make_my_own_app'),
+        text: I18n.t('footer.try_hour_of_code'),
```

Comment:
No. Applab doesn't use this _share_footer layout, because it generates its footer on the client side: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/applab/applab.js#L346-L346